### PR TITLE
Add Pulsar Community Weekly Update - 2021-02-01~ 2021-02-07

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2021/2021-02/2021-02-08-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+- [TGIP] Weekly live stream about Pulsar and its ecosystem.
+
+    - 019: Latest Updates On Apache Pulsar
+  
+        - https://t.co/zXjaokSGKa?amp=1
+
+### Blog / Article
+
+- How to choose pulsar vs Kafka? -- Jesse Anderson
```
Comment (Anonymitaet) [inline content/2021/2021-02/2021-02-08-pulsar-weekly.md:215]:
```suggestion
- How to choose Pulsar vs Kafka? -- Jesse Anderson
```

**Review by Anonymitaet — APPROVED** [review]

File: content/2021/2021-02/2021-02-08-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+- [Client] Add default implementation of `CryptoKeyReader`.
+
+    https://github.com/apache/pulsar/pull/9379 (@[massakam](https://github.com/massakam))
+
+- [BookKeeper] Support configuring the BookKeeper `Opportunistic Striping` feature.
+
+    https://github.com/apache/pulsar/pull/9232 (@[eolivelli](https://github.com/eolivelli))
+
+- [Admin] Support getting the applied policy for `MaxConsumers`.
```
Comment (BewareMyPower) [inline content/2021/2021-02/2021-02-08-pulsar-weekly.md:43]: `applied` -> `topic applied`

File: content/2021/2021-02/2021-02-08-pulsar-weekly.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+- [Functions] Enhance the Kubernetes manifest customizer with default options.
+
+    https://github.com/apache/pulsar/pull/9445 (@[freeznet](https://github.com/freeznet))
+
+- [Managed Ledger] Support configuring the `ManagedLedger` storage.
+
+    https://github.com/apache/pulsar/pull/9397 (@[sijie](https://github.com/sijie))
+
+- [Admin] Ensure that the admin operations on Statestore are not blocked.
```
Comment (BewareMyPower) [inline content/2021/2021-02/2021-02-08-pulsar-weekly.md:99]: `not blocked` -> `non-blocking`

File: content/2021/2021-02/2021-02-08-pulsar-weekly.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+    https://github.com/apache/pulsar/pull/9348 ([@pkumar-singh](https://github.com/pkumar-singh))
+
+- [C++] Remove the Boost::System runtime dependency.
+
+    https://github.com/apache/pulsar/pull/9498 (@[merlimat](https://github.com/merlimat))
+
+- [CLI] Update the `-cluster` parameter in t`he delete-cluster-metadata` command to `--cluster`.
```
Comment (BewareMyPower) [inline content/2021/2021-02/2021-02-08-pulsar-weekly.md:107]: the `delete-cluster-metadata`

File: content/2021/2021-02/2021-02-08-pulsar-weekly.md [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+
+    https://github.com/apache/pulsar/pull/9487 (@[yangl](https://github.com/yangl))
+
+- [Topic Policy] Support getting the applied policy for retention.
```
Comment (BewareMyPower) [inline content/2021/2021-02/2021-02-08-pulsar-weekly.md:111]: `applied` -> `topic applied`

File: content/2021/2021-02/2021-02-08-pulsar-weekly.md [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+
+    https://github.com/apache/pulsar/pull/9362 (@[315157973](https://github.com/315157973))
+
+- [Topic Policy] Support getting the applied policy for the offloader.
```
Comment (BewareMyPower) [inline content/2021/2021-02/2021-02-08-pulsar-weekly.md:115]: `applied` -> `topic applied`

File: content/2021/2021-02/2021-02-08-pulsar-weekly.md [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+
+- [Flaky Test] Fix the `MessagePublishBufferThrottleTest` Flaky test.
+
+    https://github.com/apache/pulsar/pull/9376 (@[Renkai](https://github.com/Renkai))
+
+- [Flaky Test] Fix the `TopicReaderTest.testMultiReaderIsAbleToSeekWithTimeOnMiddleOfTopic` Flaky test.
+
+    https://github.com/apache/pulsar/pull/9375 (@[Renkai](https://github.com/Renkai))
+
+- [C++] Fix the issue that pulsar-client-java does not set `replicator_to` into messages when batching is enabled.
```
Comment (BewareMyPower) [inline content/2021/2021-02/2021-02-08-pulsar-weekly.md:129]: `Fix broken replicated msg to specific cluster`.

BTW, if you think the original description is not clear, you can just change `pulsar-client-java` here to `C++ client` or `pulsar-client-cpp`.

**Comment by BewareMyPower** [comment]
I've left some comments, PTAL.

**Pull request merged**